awesome_dashboard/static/src/dashboard/dashboard.js

```diff
@@ -3,27 +3,43 @@ import { registry } from "@web/core/registry";
 import { Layout } from "@web/search/layout";
 import { useService } from "@web/core/utils/hooks";
 import { DashboardItem } from "./dashboard_item";
-import { PieChart } from "./pie_chart/pie_chart";
-import { items } from "./dashboard_items";
+import { ConfigDialog } from "./config_dialog";
+import { browser } from "@web/core/browser/browser";
 import { _t } from "@web/core/l10n/translation";
 
 class AwesomeDashboard extends Component {
     static template = "awesome_dashboard.AwesomeDashboard";
-    static components = { Layout, DashboardItem, PieChart }
+    static components = { Layout, DashboardItem }
     setup() {
         this.action = useService("action");
         this.result = useState(useService("awesome_dashboard.statistics"));
         this.items = registry.category("awesome_dashboard").getAll();
-
+        this.dialog = useService("dialog");
+        // retreive disabled items
```

Comment:
```suggestion
        // Retrieve disabled items
```
🥇 I always love doing this. I suggest downloading the 'Code Spell Checker' extension on VsCode.